### PR TITLE
ci(dependabot): update assignees and reviewers

### PR DESCRIPTION
**Pull request by lolottetheclash**
Self explanatory.

### CODE REVIEW

**Review by amimart — APPROVED** [review]
🥳

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,13 +5,14 @@ updates:
     schedule:
       interval: weekly
     reviewers:
-      - amimart
       - ccamel
       - fredericvilcot
       - ErikssonJoakim
+      - lolottetheclash
```
Comment (fredericvilcot) [inline .github/dependabot.yml:11]: I will have added the whole team as reviewers to keep the consistency about updates knowledge 😌

**Review by ccamel — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -21,9 +22,11 @@ updates:
...
     assignees:
       - ccamel
       - fredericvilcot
+      - lolottetheclash
     open-pull-requests-limit: 5
   - package-ecosystem: docker
```
Comment (fredericvilcot) [inline .github/dependabot.yml:31]: If you don't mind, could please force `dependabot` configuration to only open PR about `docker` related to patches? Thanks 😎

**Review by fredericvilcot — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (ccamel) [reply .github/dependabot.yml:33]: @fredericvilcot I prefer to keep Dependabot PRs for all Docker images. This way, we can stay up-to-date with all available upgrades and security patches, especially regarding the `nginx` docker image we use.

Comment (ccamel) [reply .github/dependabot.yml:33]: All right. Thanks.

**Pull request merged**